src/components/ButtonPanel.js

```diff
@@ -7,26 +8,30 @@ const nums3 = ['4', '5', '6', '-'];
 const nums4 = ['1', '2', '3', '+'];
 const nums5 = ['0', '.', '='];
 
-const ButtonPanel = () => (
+const ButtonPanel = ({ clickHandler }) => (
   <div className="display">
     <>
       {nums1.map((num) => (
```

Comment:
Kindly implement the handleClick function in the ButtonPanel component, instead of passing it down from the parent component